prebid-mobile/prebid-mobile.md

```diff
@@ -11,31 +11,93 @@ nav_section: prebid-mobile
 
-This means you **must** have a Prebid Server account in order to use Prebid Mobile. To set up your Prebid Server account for Prebid Mobile, refer to [Get Started with Prebid Server]({{site.github.url}}/prebid-mobile/prebid-mobile-pbs.html)
+## How It Works
 
- 1. As the Prebid Mobile module runs, it fetches bids from your demand partners integrated with Prebid Server.
- 2. Our Prebid Mobile module sends this bid information to your primary ad server SDK using key/value targeting.
- 3. The primary ad server SDK sends a request to the primary ad server including these custom parameters.  These parameters will match the key/value targeting previously configured on Prebid line items by your ad ops teams.
- 4. If the Prebid Mobile bid wins, the primary ad server returns our creative JS to your primary ad server SDK's ad view.
- 5. The creative JS in the ad view fetches the actual cached creative content from Prebid Server. When this happens, the impression is counted.
+Prebid Mobile works in conjunction with Prebid Server, demand partners, and your primary ad server to deliver programmable demand to your mobile app.
+
+### What You Need to Get Started
+
+Before you begin to use Prebid Mobile in your apps, you need to prepare your end-to-end system.
+
+**Mobile Developers**
+
+The following is a one-time setup process for *mobile developers*:
+
+-   [Set up a Prebid Server account]({{site.github.url}}/prebid-mobile/prebid-mobile-pbs.html).
+
+-   Download the Prebid Mobile SDK:
+    -   [SDK for iOS](https://github.com/prebid/prebid-mobile-ios)
+    -   [SDK for Android](https://github.com/prebid/prebid-mobile-android)
+
+After the initial setup, ad slots and line items will need to be set up and maintained:
+
+-   *Mobile developers* register ad units with the Prebid Mobile framework.
+    -   [iOS Code Integration]({{site.github.url}}/prebid-mobile/code-integration-ios.html)
+    -   [Android Code Integration]({{site.github.url}}/prebid-mobile/code-integration-android.html)
+
+**Ad Ops**
+
+-   *Ad ops team members* configure their primary ad server with Prebid Mobile line items targeted to key/values.
+    -   [Set Up Line Items for DFP]({{site.github.url}}/prebid-mobile/adops-line-item-setup-dfp.html)
+    -   [Set Up Line Items for MoPub]({{site.github.url}}/prebid-mobile/adops-line-item-setup-mopub.html)
+
+### Lifecycle of a Mobile Ad Call
+
+After you've set up Prebid Mobile and created your line items on your ad server, your ad calls go through the following workflow:
 
 {: .pb-img.pb-lg-img :}
-![How Prebid Mobile Works - Diagram]({{site.baseurl}}/assets/images/prebid-mobile/prebid-mobile.png)
+![How Prebid Mobile Works - Diagram]({{site.baseurl}}/assets/images/prebid-mobile/pbm-overview-flow.png)
+
+1.  Prebid Mobile calls Prebid Server, sending along ad unit information.
+2.  Prebid Server passes ad unit information on to the demand partners.
+3.  Each demand partner conducts an internal auction and sends the top bids to Prebid Server.
```

Comment:
Let's generalize this -- we can't assume we know about the internal auction logic for individual Prebid Server demand partners.